ferrocene/doc/evaluation-plan/src/qualification-scope.rst

```diff
@@ -5,24 +5,29 @@ Qualification Scope
 ===================
 
 The Ferrocene qualification encompasses the qualification of the
-Ferrocene compiler's functionality as identified in the
+Rust toolchain's functionality as identified in the
 :doc:`user-manual:index` and the :doc:`specification:index`
 
-The Ferrocene compiler links to libraries ``liballoc``, ``libcore``, and
+The Rust compiler, ``rustc``, links to libraries ``liballoc``, ``libcore``, and
 ``libstd``. These libraries are evaluated and tested within the scope of
 Ferrocene qualification *for compiler use only*. The use of these libraries
 by end-use code is outside the scope of the current Ferrocene
 qualification. It is the end-user responsibility to qualify these libraries if
 they are used in their code.
 
-The other development tools besides ``rustc`` are not qualified and are
-distributed for convenience only. This includes but is not limited to:
+There are other development tools besides ``rustc``.
+
+The qualified tools are:
+
+* ``rustfmt``
```

Comment:
I would probably remove the first sentence, and say the qualified tools are `rustc` and `rustfmt`.

Reply:
By first sentence you mean following?

> There are other development tools besides ``rustc``.

Reply:
Yes.